windows/Accelerometer.cs

```diff
@@ -57,22 +53,18 @@ public void startUpdates()
             {
                 _accelerometer = Sensors.Accelerometer.GetDefault();
                 if (_accelerometer == null) throw new Exception("No Accelerometer found");
+
+                this.setUpdateInterval(this.interval);
             }
             _accelerometer.ReadingChanged += new TypedEventHandler<Sensors.Accelerometer, Sensors.AccelerometerReadingChangedEventArgs>(ReadingChanged);
         }
 
         [ReactMethod]
         public void stopUpdates()
         {
-            //throw new NotImplementedException();
+            _accelerometer.ReadingChanged -= new TypedEventHandler<Sensors.Accelerometer, Sensors.AccelerometerReadingChangedEventArgs>(ReadingChanged);
```

Comment:
How does `-=` work in this context? Wouldnt the new value generate a new instance of the event handler that isn't in ReadingChanged?